LuaUI/Widgets/gui_s44Only.lua

```diff
@@ -6,7 +6,7 @@ function widget:GetInfo()
 		date = "16 January 2012",
 		license = "GNU GPL v2",
 		layer = 1,
-		enabled = true,
+		enabled = false,

```

Comment:
What if someone has it already enabled??
May we just remove this widget?

Reply:
I would be for removing this widget while there is similar logic which can be potentially used in `\LuaUI\Widgets\gui_notAchili_ss44_ui.lua` (at the bottom) which is less restrictive, killing only the most used independently distributed widgets which some legacy players can have stored in their global `SpringData/LuaUI/Widgets` folder

Reply:
IDEM... If it becomes useless... For me you can remove

Reply:
ok, removed in  [a0b5973]